frontend/src/metabase/css/core/text.css

```diff
@@ -182,7 +182,7 @@
 
 .text-wrap {
   overflow-wrap: anywhere;
-  word-break: break-word;
+  word-break: break-all;
```

Comment:
This will have unintended consequences - everywhere in Metabase, not just on accordions. I would leave that, and just remove the flex-full specifically for the accordion.
![image](https://user-images.githubusercontent.com/1447303/146805162-e5524074-5a46-4e9d-a079-a74cb1c794ab.png)
![image](https://user-images.githubusercontent.com/1447303/146805445-f6cf45dc-1eeb-47a8-9d3d-2c65f7fe90a5.png)

Reply:
@flamber Done. Hyphenated "words" will get line breaks.